include/stock_fft/heffte_stock_vec_types.h

```diff
@@ -9,7 +9,7 @@
 
 #include "heffte_config.h"
 
-#ifdef __AVX__
+#if defined(Heffte_ENABLE_AVX) || defined(Heffte_ENABLE_AVX512)
```

Comment:
AVX512 force enables the AVX, we should just use `#ifdef Heffte_ENABLE_AVX`

Reply:
Thanks, make sense :)
```suggestion
#if defined(Heffte_ENABLE_AVX)
```